dashboard/app/views/api/hoc_courses_narrow.html.haml

```diff
@@ -1,7 +1,7 @@
 .row{style: 'margin: 0px;'}
-  = render partial: 'shared/course_tall_block', locals: { id: Script::MINECRAFT_DESIGNER_NAME, showState: false }
   = render partial: 'shared/course_tall_block', locals: { id: Script::STARWARS_NAME, showState: false }
   = render partial: 'shared/course_tall_block', locals: { id: Script::FROZEN_NAME, showState:   false }
+  = render partial: 'shared/course_tall_block', locals: { id: 'hoc_encryption', showState: false }
```

Comment:
Can you add `hoc_encryption` second to last between `text_compression` and `conditionals`?